src/edu/ucsb/nceas/metacat/EventLog.java

```diff
@@ -147,7 +147,7 @@ public void log(String ipAddress, String userAgent, String principal, String doc
     	        }
     	        
             } catch (Exception e) {
-            	logMetacat.error("Could not update event index information", e);
+            	logMetacat.warn("Could not update event index information " + e.getMessage());
```

Comment:
Is WARN a good log level here or should this stay ERROR?

Reply:
I saw lots of useless error from there. So I degraded it to warn.